Cargo.toml

```diff
@@ -48,31 +48,30 @@ zeroize = { version = "1.8", features = ["zeroize_derive"] }
 
 [dev-dependencies]
 assert_matches = "1.5"
-ic-ic00-types = { git = "https://github.com/dfinity/ic", rev = "release-2023-09-27_23-01" }
-ic-base-types = { git = "https://github.com/dfinity/ic", rev = "release-2023-09-27_23-01" }
-ic-config = { git = "https://github.com/dfinity/ic", rev = "release-2023-09-27_23-01" }
-ic-crypto-test-utils-reproducible-rng = { git = "https://github.com/dfinity/ic", rev = "release-2024-09-12_01-30-base" }
-ic-state-machine-tests = { git = "https://github.com/dfinity/ic", rev = "release-2023-09-27_23-01" }
-ic-test-utilities-load-wasm = { git = "https://github.com/dfinity/ic", rev = "release-2023-09-27_23-01" }
+candid_parser = { workspace = true }
+ic-crypto-test-utils-reproducible-rng = { git = "https://github.com/dfinity/ic", rev = "release-2024-09-26_01-31-base" }
+ic-test-utilities-load-wasm = { git = "https://github.com/dfinity/ic", rev = "release-2024-09-26_01-31-base" }
 itertools = "0.13"
 maplit = "1"
+pocket-ic = "5.0.0"
 proptest = { workspace = true }
 rand = "0.8"
 
 [workspace.dependencies]
-candid = { version = "0.9" }
+candid = { version = "0.10.10" }
+candid_parser = {version = "0.1.4"}
 ethnum = { version = "1.5.0", features = ["serde"] }
 futures = "0.3.30"
 getrandom = { version = "0.2", features = ["custom"] }
 hex = "0.4.3"
-ic-cdk = "0.10"
+ic-cdk = "0.16.0"
 ic-cdk-bindgen = "0.1"
-ic-cdk-macros = "0.7"
+ic-cdk-macros = "0.16.0"
 ic-certified-map = "0.4"
 ic-metrics-encoder = "1.1"
-ic-stable-structures = "0.5"
-minicbor = { version = "0.19.1", features = ["alloc", "derive"] }
-minicbor-derive = "0.13.0"
+ic-stable-structures = "0.6.5"
```

Comment:
When building / running the tests locally on this branch, I get an `unresolved import 'candid'` error associated with this dependency. @gregorydemay, did this happen for you at some point, and if so, did you find a workaround?

Reply:
@rvanasa : That's not something I experienced (neither on devenv nor locally on Mac M1) and since CI doesn't encounter that problem and start fresh I would think that this is maybe a local problem. Did you maybe try `cargo clean`? Otherwise, happy to do a debugging session to see if we can figure out what the problem is.